src/main/java/edu/iastate/ece/sd/sdmay2126/runner/selenium/FBASeleniumInputProgrammer.java

```diff
@@ -75,8 +75,64 @@ public void programInputs(Job job, WebElement scopedFBACard) throws SeleniumIden
         activCoe.clear();
         activCoe.sendKeys(Float.toString(params.getActivationCoefficient()));
 
+        //Finds clears and sets the value of the maxCarbonUptake value
+        System.out.println("Setting max carbon uptake...");
+        WebElement carbUp = scopedFBACard
+                .findElement(By.cssSelector("div[data-parameter='max_c_uptake']"))
+                .findElement(By.cssSelector("input[class='form-control']"));
+        carbUp.clear();
+        carbUp.sendKeys(Float.toString(params.getMaxCarbonUptake()));
+
+        //Finds clears and sets the value of the maxNitrogenUptake value
+        System.out.println("Setting max nitrogen uptake...");
+        WebElement nitroUp = scopedFBACard
+                .findElement(By.cssSelector("div[data-parameter='max_n_uptake']"))
+                .findElement(By.cssSelector("input[class='form-control']"));
+        nitroUp.clear();
+        nitroUp.sendKeys(Float.toString(params.getMaxNitrogenUptake()));
+
+        //Finds clears and sets the value of the maxPhosphateUptake value
+        System.out.println("Setting max phosphate uptake...");
+        WebElement phosUp = scopedFBACard
+                .findElement(By.cssSelector("div[data-parameter='max_p_uptake']"))
+                .findElement(By.cssSelector("input[class='form-control']"));
+        phosUp.clear();
+        phosUp.sendKeys(Float.toString(params.getMaxPhosphateUptake()));
+
+        //Finds clears and sets the value of the maxSulfurUptake value
+        System.out.println("Setting max sulfur uptake...");
+        WebElement sulfUp = scopedFBACard
+                .findElement(By.cssSelector("div[data-parameter='max_s_uptake']"))
+                .findElement(By.cssSelector("input[class='form-control']"));
+        sulfUp.clear();
+        sulfUp.sendKeys(Float.toString(params.getMaxSulfurUptake()));
```

Comment:
If no value is supplied for this parameter does this still run without exceptions?